Cargo.toml

```
@@ -4,7 +4,7 @@ name = "http"
 # - Update html_root_url in lib.rs.
 # - Update CHANGELOG.md.
 # - Create git tag
-version = "0.1.20"
+version = "0.2.0-a.0"
```

I've noticed the general convention seems to be spelling this out, as in
```suggestion
version = "0.2.0-alpha.0"
```